app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -95,9 +99,29 @@ class StateFragmentPresenter @Inject constructor(
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
       logger.d("StateFragment", "getCurrentState: ${result.state.name}")
+      if (getContentListFragment() == null) {
+        val contentListFragment = ContentListFragment()
+        val args = Bundle()
+        args.putString("exploration_id", dummyExploration_id)
```

Comment:
Shouldn't we be using the same exploration ID that's being passed into StateFragmentPresenter?

If the explorations that are being included in the APK itself are not satisfactory for testing the state fragment, feel free to modify them to add additional rich text contents as needed.

Reply:
Yes

> Shouldn't we be using the same exploration ID that's being passed into StateFragmentPresenter?

Yes we could use. I added dummy exploration id for testing image and modified json.